MerlinAU.sh

```diff
@@ -6430,7 +6500,7 @@ _PostRebootRunNow_()
    fi
 
    _UserTraceLog_ "END of $logMsg [$$curWaitDelaySecs sec.]"
-   sleep 60  ## Let's wait a bit & proceed ##
+   sleep 30  ## Let's wait a bit & proceed ##
```

Comment:
Decrease back down now that we know this isn't the problem with the WAN connection/nodes.